src/components/EditAndApproveStory/EditAndApproveStory.tsx

```diff
@@ -0,0 +1,327 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Card,
+  CardMedia,
+  Divider,
+  FormControlLabel,
+  FormHelperText,
+  Grid,
+  MenuItem,
+  TextField,
+  Typography,
+  Switch,
+} from '@material-ui/core';
+import { makeStyles, Theme } from '@material-ui/core/styles';
+
+import { Button } from '../Button/Button';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  alignRight: {
+    textAlign: 'right',
+  },
+  greyLink: {
+    color: theme.palette.grey[600],
+  },
+}));
+
+export interface EditAndApproveStoryProps {
+  /**
+   * The name of the publisher
+   */
+  publisher: string;
+  /**
+   * The name of the author
+   */
+  author: string;
+  /**
+   * The URL of the story
+   */
+  url: string;
+  /**
+   * The title of the story
+   */
+  headline: string;
```

Comment:
Update to `title`
```suggestion
  title: string;
```

Reply:
All done @kkyeboah! I cleaned up all the fixup commits as well.